common/display/displayplanemanager.cpp

```diff
@@ -200,6 +201,17 @@ bool DisplayPlaneManager::ValidateLayers(
         // If we are able to composite buffer with the given plane, lets use
         // it.
         bool fall_back = FallbacktoGPU(plane, layer, commit_planes);
+
+        if (fall_back) {
+          has_gpu_fallback_layer = true;
```

Comment:
Just double checked the source code that have fall_back set to true, it will have the chance to call ResetPlaneTarget. It will ensure DisplayPlaneState object properly initialized and OverlayPlane object bundled with right DisplayPlaneState object. 
In old logic, there are chances that plane separation is needed for some layers but  DisplayPlaneState object and OverlayPlane object are not properly initialized & linked.

Will replace these code pieces with more clean purpose.